gmm.py

```diff
@@ -331,3 +411,92 @@ def __update_pi(self, pi):
         assert pi.size() in [(1, self.n_components, 1)], "Input pi does not have required tensor dimensions (%i, %i, %i)" % (1, self.n_components, 1)
 
         self.pi.data = pi
+
+        
+    @staticmethod
```

Comment:
I realize this was used in testing, but is `@staticmethod` still required or can this be removed?